src/components/Calculator.js

```diff
@@ -1,37 +1,50 @@
         <div className="calculator">
-          <button type="button" className="btn">AC</button>
-          <button type="button" className="btn">+/-</button>
-          <button type="button" className="btn">%</button>
-          <button type="button" className="orangeBtn">÷</button>
-          <button type="button" className="btn">7</button>
-          <button type="button" className="btn">8</button>
-          <button type="button" className="btn">9</button>
-          <button type="button" className="orangeBtn">x</button>
-          <button type="button" className="btn">4</button>
-          <button type="button" className="btn">5</button>
-          <button type="button" className="btn">6</button>
-          <button type="button" className="orangeBtn">-</button>
-          <button type="button" className="btn">1</button>
-          <button type="button" className="btn">2</button>
-          <button type="button" className="btn">3</button>
-          <button type="button" className="orangeBtn">+</button>
-          <button type="button" className="btn-0">0</button>
-          <button type="button" className="btn">.</button>
-          <button type="button" className="orangeBtn">=</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>AC</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>+/-</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>%</button>
+          <button type="button" className="orangeBtn" onClick={this.updateCalculator}>÷</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>7</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>8</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>9</button>
+          <button type="button" className="orangeBtn" onClick={this.updateCalculator}>x</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>4</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>5</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>6</button>
+          <button type="button" className="orangeBtn" onClick={this.updateCalculator}>-</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>1</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>2</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>3</button>
+          <button type="button" className="orangeBtn" onClick={this.updateCalculator}>+</button>
+          <button type="button" className="btn-0" onClick={this.updateCalculator}>0</button>
+          <button type="button" className="btn" onClick={this.updateCalculator}>.</button>
+          <button type="button" className="orangeBtn" onClick={this.updateCalculator}>=</button>
```

Comment:
[OPTIONAL]

- I think it will be better if you create a reusable button component that you will use throughout your app (one of the beauties of React 😄) instead of writing too many buttons which means repeating yourself. Please consider this 👍🏽

Reply:
Noted Thanks